lib/mapObjects/CBank.cpp

```diff
@@ -275,6 +275,10 @@ void CBank::doVisit(const CGHeroInstance * hero) const
 	//grant resources
 	if (bankConfig)
 	{
+		// add statistics
+		if(ID.toEnum() == Obj::DRAGON_UTOPIA)
+			cb->gameState()->statistic.accumulatedValues[hero->getOwner()].lastRansackingDragonUtopiaDay = cb->gameState()->getDate(Date::DAY);
+
```

Comment:
Perhaps use some more generic trigger to handle banks from mods? E.g. banks above certain 'value'? Or instead collect total value of cleared banks? Utopias would be clearly visible in it anyway. Or anything else that is not check for H3 object ID?